config/ftbquests/normal/chapters/6c29cdf5/1bb0a209.snbt

```diff
@@ -0,0 +1,25 @@
+{
+	title: "{q.botania.the_spectator.name}",
```

Comment:
Its not an issue, but... Did you really manually renamed all quest descriptions for new quests? Or you figure out how to use JS script?

Reply:
Manually